src/main/java/core/basesyntax/CountFigure.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class CountFigure {
+    public static final int CountFigure = 5;
```

Comment:
```suggestion
    public static final int COUNT_FIGURE = 5;
```
we declare constants with upper case